### PR TITLE
Clean checkin of Audit docs.

### PR DESCRIPTION
**Pull request by tbedford**
## Description

Clean checkin of Audit docs. This should be considered the initial draft of the documentation for the Audit API (Beta version). Docs are flagged as Beta.

> This replaces #867 which was messy.

- [x] DO NOT MERGE UNTIL OLIVER HAS APPROVED

### CODE REVIEW

File: _documentation/audit/building-blocks/get-events-with-filtering.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+navigation_weight: 3
+---
+
+# Get audit events with filtering
+
+In this building block you see how to get a list of audit events with filtering.
+
+## Parameters
+
+You can filter the list of event objects returned by using query parameters. The parameters than can be specified are shown in the following table:
```
Comment (marklewin) [inline _documentation/audit/building-blocks/get-events-with-filtering.md]: "The parameters than can be specified" -> "The parameters that can be specified"

File: _documentation/audit/overview.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+* [Guides](#guides)
+* [Building Blocks](#building-blocks)
+* [Tutorials](#tutorials)
+* [Reference](#reference)
+
+## Concepts
+
+### Authentication
+
+Interactions with the Audit API are authenticated using Basic Authentication. Basic Authentication allows you to use your `NEXMO_API_KEY` and `NEXMO_API_SECRET` to validate your API requests. For more information see [authenticating your applications](/concepts/guides/authentication).
```
Comment (marklewin) [inline _documentation/audit/overview.md]: "For more information see authenticating your applications." The "For more ...see" suggests to me that 'authenticating your applications' links to an article of the same name, but the target page is called 'Authentication'. Perhaps use something like "Learn how to authenticate your application" as the link text?

(Regarding authentication, the docs suggest that I have to send the api key and secret base64-encoded for the newer APIs, but I didn't have to do this.)

File: _documentation/audit/overview.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+* Account settings updates.
+* A Nexmo Number gets assigned to an application.
+* Creation of a Nexmo application.
+
+Further information on types of audit events is given in the [Audit Event Guide](/audit/guides/audit-events).
+
+### Audit Event object
+
+An example audit event object is 'updating a number' which has `event_type` of `NUMBER_UPDATED`:
```
Comment (marklewin) [inline _documentation/audit/overview.md]: The example JSON doesn't really demonstrate an audit event object, but just the `context` property of that object. Perhaps explain that this is just the field within the event object that describes what has changed?

File: _tutorials/retrieve-audit-events.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
+---
+title: Retrieve audit events
+products: audit
+description: "You can retrieve filtered lists of audit events. Audit events log activity in a Nexmo account."
+languages:
+    - Curl
+---
+
+# Audit - Retrieve audit events
```
Comment (marklewin) [inline _tutorials/retrieve-audit-events.md]: The title of the tutorial is "Audit - Retrieve Audit Events", but tutorials for the other APIs don't precede the title of the tutorial with the name of the API.

File: _tutorials/retrieve-audit-events.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+It is possible to retrieve a record of all audit events associated with your Nexmo account. You can also filter this list based on dates, keyword, user and event type.
```
Comment (marklewin) [inline _tutorials/retrieve-audit-events.md]: "It is possible to" -> "You can"

File: _tutorials/retrieve-audit-events.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+You can narrow this down further in various ways. For example you can also filter based on [audit event type](/audit/guides/audit-events#audit-event-types).
+
+So for example, to find audit events in August of type `NUMBER_ASSIGN` you could enter the following:
+
+```
+$  curl "https://api.nexmo.com/beta/audit/events?date_from=2018-08-01&date_to=2018-08-31&event_type=NUMBER_ASSIGN" \
+     -u 'NEXMO_API_KEY:NEXMO_API_SECRET'
+```
+
+You can further filter based on `search_text`. For example to find all audit events contaning the text "password" you can enter the following command:
```
Comment (marklewin) [inline _tutorials/retrieve-audit-events.md]: 'containing' is misspelled.

File: _tutorials/retrieve-audit-events.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+
+This would then return the audit event object JSON for the specified audit event.
+
+## Conclusion
+
+Using the filtering capabilities of the Audit API you have complete control regarding the audit events that you retrieve.
+
+## Resources
+
+* [Audit API docs](/audit)
```
Comment (marklewin) [inline _tutorials/retrieve-audit-events.md]: The link to Audit API docs does not work. Should it point to /audit/overview?

File: _documentation/audit/overview.md (hunk 2 above)
Comment (tbedford) [reply _documentation/audit/overview.md]: Changed link text. Also added Audit to Authentication page table.

If you are using Curl the `-u` option does the Base64 encoding for you (although the Curl man page doesn't explicitly say that).

File: _documentation/audit/overview.md (hunk 3 above)
Comment (tbedford) [reply _documentation/audit/overview.md]: Copy and paste error. Corrected.

File: _documentation/audit/building-blocks/get-events-with-filtering.md (hunk 1 above)
Comment (tbedford) [reply _documentation/audit/building-blocks/get-events-with-filtering.md]: Fixed.

File: _documentation/audit/guides/audit-events.md [hunk 8]
```diff
@@ -0,0 +1,79 @@
+---
+title: Audit Events
+navigation_weight: 1
+description: A guide to Audit Events.
+---
+
+# Audit Events
+
+Audit Events are generated automatically by Nexmo as the user interacts with either the Nexmo API or the Nexmo Dashboard. Each Audit Event object has a type and assocaited metadata.
```
[the remark this answers is not in the document]
Comment (tbedford) [reply _documentation/audit/guides/audit-events.md]: Fixed.

File: _documentation/audit/overview.md [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+# Audit API Overview
+
+The Nexmo Audit API allows you to track events. These events are activities created by using the API or by interacting with the Nexmo Dashboard. The Audit API allows you to monitor your Nexmo account.
+
+With this API you can:
+
+* Retrieve a list of Audit events.
+* Retrieve a specific Audit event.
+
+It is possible to filter the events retrieved based on parameters such as date, user ID, and event type.
```
[the remark this answers is not in the document]
Comment (tbedford) [reply _documentation/audit/overview.md]: Fixed.

File: _documentation/audit/overview.md [hunk 10]
```diff
@@ -0,0 +1,102 @@
+---
+title: Overview
+navigation_weight: 1
+description: The Audit API overview.
+---
+
+# Audit API Overview
+
+The Nexmo Audit API allows you to track events. These events are activities created by using the API or by interacting with the Nexmo Dashboard. The Audit API allows you to monitor your Nexmo account.
```
[the remark this answers is not in the document]
Comment (tbedford) [reply _documentation/audit/overview.md]: Prefer your wording! Fixed.

File: _documentation/audit/overview.md [hunk 11]
```diff
@@ -0,0 +1,102 @@
...
+
+It is possible to filter the events retrieved based on parameters such as date, user ID, and event type.
+
+## Beta
+
+This API is currently in Beta.
+
+Nexmo always welcomes your feedback. Your suggestions help us improve the product. If you do need help, please email [support@nexmo.com](mailto:support@nexmo.com) and include Audit API in the subject line. Please note that during the Beta period support times are limited to Monday to Friday.
+
+During Beta Nexmo may expand the capabilities of the Audit API.
```
[the remark this answers is not in the document]
Comment (tbedford) [reply _documentation/audit/overview.md]: Fixed.

File: _tutorials/retrieve-audit-events.md (hunk 4 above)
Comment (tbedford) [reply _tutorials/retrieve-audit-events.md]: Well spotted! Fixed.

File: _tutorials/retrieve-audit-events.md (hunk 5 above)
Comment (tbedford) [reply _tutorials/retrieve-audit-events.md]: Fixed.

File: _tutorials/retrieve-audit-events.md (hunk 6 above)
Comment (tbedford) [reply _tutorials/retrieve-audit-events.md]: Fixed.

File: _tutorials/retrieve-audit-events.md (hunk 7 above)
Comment (tbedford) [reply _tutorials/retrieve-audit-events.md]: Yes it should be! Fixed.

File: _documentation/audit/overview.md (hunk 2 above)
Comment (marklewin) [reply _documentation/audit/overview.md]: Right! Didn't know that. Thanks :)

File: _documentation/audit/overview.md [hunk 12]
```diff
@@ -0,0 +1,102 @@
...
+## Building Blocks
+
+* [Get Audit events](/audit/building-blocks/get-events)
+* [Get Audit event](/audit/building-blocks/get-event)
+* [Get Audit events with filtering](/audit/building-blocks/get-events-with-filtering)
+* [Get Audit event types](/audit/building-blocks/get-event-types)
+
+## Tutorials
+
+* [Tutorials](/audit/tutorials)
```
[the remark this answers is not in the document]
Comment (mheap) [reply _documentation/audit/overview.md]: Could we use the tutorial list filter here?

[the remark this answers is not in the document]
Comment (tbedford) [reply _documentation/audit/overview.md]: Yes, latest commit using the new constructs for Guides, Building Blocks and Tutorials.

**Pull request merged**